src/Avalonia.X11/X11Platform.cs

```diff
@@ -96,17 +97,19 @@ public void Initialize(X11PlatformOptions options)
             if (options.UseGpu)
             {
                 if (options.UseEGL)
-                    EglPlatformOpenGlInterface.TryInitialize();
+                    EglPlatformGraphics.TryInitialize();
                 else
-                    GlxPlatformOpenGlInterface.TryInitialize(Info, Options.GlProfiles);
+                    GlxPlatformGraphics.TryInitialize(Info, Options.GlProfiles);
             }
 
-            var gl = AvaloniaLocator.Current.GetService<IPlatformOpenGlInterface>();
+            var gl = AvaloniaLocator.Current.GetService<IPlatformGraphics>();
             if (gl != null)
-                AvaloniaLocator.CurrentMutable.Bind<IPlatformGpu>().ToConstant(gl);
+                AvaloniaLocator.CurrentMutable.Bind<IPlatformGraphics>().ToConstant(gl);
```

Comment:
That's a weird code. You read IPlatformGraphics from the locator and set it back if it's not null?